### ISSUE TITLE
Use the latest stable Optimism release

### ISSUE THREAD

**Issue by prathamesh0** [cerc-io/stack-orchestrator#363]
A new stable [release](https://github.com/ethereum-optimism/optimism/releases) for Optimism is available; we need to move to that and hopefully it addresses some of the stability issues (see #361)

Also, the [guide](https://stack.optimism.io/docs/build/getting-started/) followed while creating the setup has been updated to run an additional `op-proposer` service, which periodically submits new L2 state roots to L1.

- [x] Upgrade Optimism to a stable release
- [x] Add an `op-proposer` service to the setup

**Comment by prathamesh0** [comment]
On starting with a fresh L2 setup, the `op-proposer` runs into an error after some time without making any proposals:
```bash
t=2023-04-24T11:20:14+0000 lvl=info msg="Initializing L2 Output Submitter"
t=2023-04-24T11:20:14+0000 lvl=info msg="Connected to L2OutputOracle"      address=0x48D0D9d2741bF7AE8bc98D53D5568d1b00D616a1 version=1.2.0
t=2023-04-24T11:20:14+0000 lvl=info msg="Starting L2 Output Submitter"
t=2023-04-24T11:20:14+0000 lvl=info msg="L2 Output Submitter started"
t=2023-04-24T11:20:26+0000 lvl=info msg="proposer submission interval has not elapsed" currentBlockNumber=0 nextBlockNumber=120
t=2023-04-24T11:20:38+0000 lvl=info msg="proposer submission interval has not elapsed" currentBlockNumber=0 nextBlockNumber=120
t=2023-04-24T11:20:50+0000 lvl=info msg="proposer submission interval has not elapsed" currentBlockNumber=0 nextBlockNumber=120
.
.
.
t=2023-04-24T11:26:14+0000 lvl=eror msg="failed to fetch output at block %d: %w" LOG15_ERROR=
t=2023-04-24T11:26:26+0000 lvl=eror msg="failed to fetch output at block %d: %w" LOG15_ERROR=
t=2023-04-24T11:26:38+0000 lvl=eror msg="failed to fetch output at block %d: %w" LOG15_ERROR=
.
.
.
```
On `l2-geth`, we see a warning:
```bash
INFO [04-24|11:26:13.014] Starting work on payload                 id=0x6b3470817c921307
INFO [04-24|11:26:13.018] Updated payload                          id=0x6b3470817c921307 number=507 hash=968baa..f95b6e txs=1 gas=49641 fees=0 root=3205f8..827cd5 elapsed=2.104ms
WARN [04-24|11:26:14.917] Served eth_getProof                      conn=172.29.0.2:55178 reqid=3535 duration="290.531µs" err="missing trie node 7c1ccb45a260eaf7a117516d56a72b7f4b38db2f8733696d8b90c344779776ba (path ) <nil>"
INFO [04-24|11:26:14.951] Stopping work on payload                 id=0x6b3470817c921307 reason=delivery
INFO [04-24|11:26:14.956] Imported new potential chain segment     number=507 hash=968baa..f95b6e blocks=1 txs=1 mgas=0.050 elapsed=1.555ms     mgasps=31.918  dirty=508.46KiB
INFO [04-24|11:26:14.959] Chain head was updated                   number=507 hash=968baa..f95b6e root=3205f8..827cd5 elapsed="201.625µs"
INFO [04-24|11:26:15.010] Starting work on payload                 id=0x24077b874849c3b8
```

**Comment by prathamesh0** [comment]
In the updated [tutorial / guide](https://stack.optimism.io/docs/build/getting-started), op-geth is run with `--gcmode=archive` instead of `--gcmode=full` as done previously. The above error doesn't occur when using `archive` mode.
However, we still keep getting the `msg="proposer submission interval has not elapsed" currentBlockNumber=0 nextBlockNumber=120` log without any proposals.

**Comment by prathamesh0** [comment]
> However, we still keep getting the msg="proposer submission interval has not elapsed" currentBlockNumber=0 nextBlockNumber=120 log without any proposals.

This issue has been addressed in the latest Optimism release [v1.0.4](https://github.com/ethereum-optimism/optimism/releases/tag/v1.0.4)
